### DISCUSSION TITLE
Understanding The First Metamorphosis: KELVIN

### DISCUSSION THREAD

**Discussion by josePereiro** [josePereiro/TankeFactory#3] [Discussions]
A first quantitative expression as consequence of Carnot's analysis can be written as (eq numbers from the main text):

$$\tag{9} Q_1/T_1 + Q_2/T_2 \le 0$$

where both $Q_1$ and $Q_2$ are heat flows from both reservoirs (see diagram). For convenience, heat flows from the cold reservoir ($T_1 > T_2$) are taken as negative.

<p align="center">
   <img src="https://github.com/josePereiro/reading_jaynesEvolutionCarnotPrinciple1988/assets/44244693/78a5aeb4-6c05-44e6-930b-4b583611190d" width=40% height=40%>
</p>

Later in the text a generalization is proposed, I quote: **A simple generalization is then obvious; we may consider a more complicated heat engine that runs cyclically, making contact successively with $n$ reservoirs at temperatures $(T_1, \dots, T_n)$. Then the 1rst law is $W = \sum Q_i$, and Carnot's principle becomes**

$$\tag{10} \sum Q_i/T_i \le 0$$

At first I was confused on how to understand this "obvious" generalization. The "more complex engine" description really created more questions than answers. For instance: 
- What is the meaning of it running cyclically? The complex engine is a reversible engine?. But, it can not be, otherwise eq. 10 will always be an equality and not general at all. 
- For eq. 10 to be negative it must have negative terms, but, how do we determine which $Q's$ are negatives? 
- What is the relations between the $T's$? Are they sorted in any way?
- etc

The only way I have been able to make sense of eq. 10 is by considering it as the result of adding several eq. 9s together. In other words, eq 9 can be generalized to use a list of pairs of reservoirs ($T_1, T_2$). By reshaping this list of pairs into a continuous one, we might arrive at a scenario like eq. 10. However, I am unable to map this into the description of the 'complex engine' mentioned above. In esense, I can't imaging a diagram (similar to the one above) for this "complex engine".

Understanding this is important for the The Second Metamorphosis.

Any thoughts?

**Comment by ta-rekt** [comment]
• I think the meaning of it running cyclically is just that you put the engine in contact with a sequence of reservoirs and then repeat the same sequence. It doesn't have to be reversible. You can think of it as changing the temperature of a reservoir continuously with a dial, always returning to the temperature you started.
• I don't think you need to determine which ones are negative, this result is just about their sum being negative. of course there will be negative $Q$'s if you change the temperature in a cyclical way, because the engine will need to spit out to the colder 'pieces' of the cycle the heat it absorbed in the hotter ones.
• not necessarily. any sequence of T's will lead to eq. 10 being true, as long as you repeat it.

What's not obvious to me is why the change of variables he did makes equation (5) 'obvious'. Also do you know what relation of probability theory he is referring to?

**Reply by josePereiro** [reply]
> • I think the meaning of it running cyclically is just that you put the engine in contact with a sequence of reservoirs and then repeat the same sequence. It doesn't have to be reversible. You can think of it as changing the temperature of a reservoir continuously with a dial, always returning to the temperature you started.

I'm still failing to understand the need of a cyclic process. Actually, I don't know what is precisely cyclic. Let say I have an engine $E$ and I make it interact sequentially with a set of reservoirs with temperature $T \in (T_1, \dots, T_i, \dots, T_N)$, as shown in the scheme. 

![image](https://github.com/josePereiro/reading_jaynesEvolutionCarnotPrinciple1988/assets/44244693/64ba6604-2827-40a5-a78d-4027c3d8f5f8)

Till now nothing cyclic is happening, but the Kelvin sum $\sum Q_i/ T_i \le 0$ can be perfectly defined. If now I run the engine interacting with the reservoirs in reverse order $T \in (T_N, \dots, T_i, \dots, T_1)$, Why is this relevant? Is it the cyclic aspect of the "complex engine"?

**30 minutes later...** Mmm, I might just figure out. The reversible aspect is not relevant for the equation  $\sum Q_i/ T_i \le 0$, it is relevant precisely because this equation holds for all processes, including cycles, and cycles are useful for other reasons, like running real engines.

**Reply by josePereiro** [reply]
___
> What's not obvious to me is why the change of variables he did makes equation (5) 'obvious'. 

![image](https://github.com/josePereiro/reading_jaynesEvolutionCarnotPrinciple1988/assets/44244693/1f667285-e839-4836-add0-e1794acaf61a)
see also [caticha, page 55](http://arxiv.org/abs/0808.0012)

___
> Also do you know what relation of probability theory he is referring to?

I think is: p(A or B) = p(A) + p(B) - p(A and B).

**Reply by ta-rekt** [reply]
awesome. I see. thanks for this!
